src/main/java/cofh/thermaldynamics/duct/entity/Camera.java

```diff
@@ -17,7 +17,7 @@ public Camera() {
 		super(null);
 		this.width = 0;
 		this.height = 0;
-		invulnerable = true;
+		this.setEntityInvulnerable(true);
```

Comment:
The overhead during gameplay should be irrelevant in this case, in a few cases the JIT will even optimize the  call away and replace it with setting the value directly.